### ISSUE TITLE
provide different env.yml file for local installation without cupy and cudatoolkit

### ISSUE THREAD

**Issue by wikfeldt** [ENCCS/HPDA-Python#58]

**Comment by qianglise** [comment]
i still did not get  why they have errors about cupy and cudatoolkit. Maybe they are using Windows?
In general, there should not be problems installing them.
In the beginning, I planned to have cupy examples, that is why it is there. 
cudatoolkit is needed for numba anyhow, so one should not remove.

**Comment by qianglise** [comment]
maybe we could make one yml file for cpu, then people could use that to install locally

**Comment by wikfeldt** [comment]
yeah i think the failure to install cupy/cudatoolkit was on windows. But i agree that having two env.yml files, one for CPU and one for GPU, makes sense.
But we might want to consider larger changes, like splitting the material into CPU and GPU parts. This is for a later discussion
